code/modules/mob/living/living_defines.dm

```diff
@@ -120,3 +120,15 @@
 	var/slowed_by_drag = TRUE //Whether the mob is slowed down when dragging another prone mob
 
 	var/is_busy = FALSE //Used for random actions that take time. ex: curbstomping. We need to make sure we can only do one of these at a time.
+
+	var/obj/chattext/chattext = new
+	
+	var/last_heard_name = null
+	var/last_used_color = null
+
+/obj/chattext

```

Comment:
You could've at least stolen proper cleanup from https://github.com/goonstation/goonstation/blob/dcedf99f66818be98991eff8dd3cd215ae4a3d37/code/WorkInProgress/pali.dm#L160